src/tfr_control/src/robot_interface.cpp

```diff
@@ -71,7 +71,7 @@ namespace tfr_control
 
         //LEFT_TREAD
         position_values[static_cast<int>(Joint::LEFT_TREAD)] = 0;
-        velocity_values[static_cast<int>(Joint::LEFT_TREAD)] = reading_a.tread_left_vel;
+        velocity_values[static_cast<int>(Joint::LEFT_TREAD)] = -reading_a.tread_left_vel;
```

Comment:
This seems like an important change...

Why would this be required?

Reply:
Counterclockwise is positive, clockwise is negative, to make that match up with our right angle gearboxes you flip them